modules/custom-post-types/comics.php

```diff
@@ -170,15 +170,19 @@ public function bulk_edit_notices() {
 	}
 
 	public function register_scripts() {
-		wp_enqueue_style( 'jetpack-comics-style', plugins_url( 'comics/comics.css', __FILE__ ) );
+		// These assets change very infrequently, don't base it on JETPACK_VERSION.
+		$version = '2020-06-11';
+
+		wp_enqueue_style( 'jetpack-comics-style', plugins_url( 'comics/comics.css', __FILE__ ), array(), $version );
```

Comment:
I'd be fine using `JETPACK_VERSION` here as well. As you wrote, it wouldn't really be necessary to update those assets' versions every month, but by keeping it simple we make sure we won't forget to update that number whenever we make changes to that file.

Reply:
@jeherve Sounds good, I'll use the constant throughout. 👍